config/prod/ubuntu-rstudio-ami.yaml

```diff
@@ -22,7 +22,7 @@ parameters:
   # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
   VolumeSize: "500"
   # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
-  AMIId: "ami-061a59e27c8da0b93"
+  AMIId: "ami-0226a8af83fcecb43"
```

Comment:
changing the AMI will wipe data from your instance. 
https://sagebionetworks.jira.com/wiki/spaces/IT/pages/810090512/Sage+Provisioner+FAQ

are you sure you want to do this?

Reply:
Yes, Currently I dont have any data on the instance. Will migrate my data there after the change.